proto/pocket/migration/tx.proto

```diff
@@ -131,9 +134,11 @@ message MsgClaimMorseApplication {
 
   // The hex-encoded address of the Morse account whose balance will be claimed.
   // E.g.: 00f9900606fa3d5c9179fc0c8513078a53a2073e
-
   string morse_src_address = 2 [(gogoproto.jsontag) = "morse_src_address"];
 
+  // The ed25519 public key of the morse account with morse_src_address.
+  bytes morse_public_key = 5 [(gogoproto.casttype) = "crypto/ed25519.PublicKey"];
```

Comment:
Why do we have `morse_public_key` AND `morse_src_address` if one can be derived from the other?

Feels like we have "two sources of truth"

Reply:
I chose to leave the address in the messages to keep the transparency in onchain events. While we can derive the address from the public key in our custom events, there are many built-in events that cosmos-sdk will generate which would lose the Morse address as a result.

Reply:
1. Can you #PUC for each `morse_public_key` with that explanation?
2. Add that explanation next to the validation function as well.
3. Please bias to bullet points.

Reply:
> there are many built-in events that cosmos-sdk

Can you provide an example?

Reply:
Moving conversation to discord.